pkg/apis/messaging/config/channel_defaults_test.go

```diff
@@ -96,7 +96,7 @@ func TestChannelDefaultsConfiguration(t *testing.T) {
 			},
 			ClusterDefault: &ChannelTemplateSpec{
 				TypeMeta: v1.TypeMeta{
-					APIVersion: "messaging.knative.dev/v1beta1",
+					APIVersion: "messaging.knative.dev/v1",
```

Comment:
I did those already. On this file - sorry (see: https://github.com/knative/eventing/pull/5314)